app/controllers/api/applications_controller.rb

```diff
@@ -19,6 +19,8 @@ class Api::ApplicationsController < FrontendController
 
   sublayout 'api/service'
 
+  helper_method :presenter
+
   def index; end
 
   def new; end
```

Comment:
Related to https://github.com/3scale/porta/pull/2522/files#r668873445

```suggestion
  def new
    @presenter = Api::ApplicationsNewPresenter.new(provider: current_account,
                                                   service: @service,
                                                   user: current_user,
                                                   cinstance: @cinstance)
  end
```

And you can get rid of the helper method. IMO, it's more straightforward.